app/src/main/java/com/example/caregiver/my_caregivee.java

```diff
@@ -0,0 +1,268 @@
+                        caregiveeIDs.add(id);
+                        caregiveeIDtoNameMap.put(id, name);
+                    }
+                    notifyDataSetChanged();
+
+                }@Override
+                public void onCancelled(DatabaseError databaseError) {
+                    Log.e("FAIL", "getCaregivees:onCancelled", databaseError.toException());
+                }
+            };
+            caregivees.addValueEventListener(valueEventListener);
+            this.caregiverID = caregiverID;
+        }
+
+
+        @Override
+        public int getGroupCount() {
+            return caregiveeIDs.size();
+        }
+
+        @Override
+        public int getChildrenCount(int i) {
+            return children[i].length;
+        }
+
+        @Override
+        public Object getGroup(int i) {
+            return caregiveeIDtoNameMap.get(caregiveeIDs.get(i));
+        }
+
+        @Override
+        public Object getChild(int i, int i1) {
+            return children[i][i1];
+        }
+
+        @Override
+        public long getGroupId(int i) {
+            return i;
+        }
+
+        @Override
+        public long getChildId(int i, int i1) {
+            return i1;
+        }
+
+        @Override
+        public boolean hasStableIds() {
+            return true;
+        }
+
+        @Override
+        public View getGroupView(int i, boolean b, View view, ViewGroup viewGroup) {
+
+            TextView textView = new TextView(my_caregivee.this.getActivity());
+            //Initialize and assign
+            textView.setText(getGroup(i).toString());
+            //Initialize string
+            String sGroup = String.valueOf(getGroup(i));
+            //Set text on text view
+            textView.setText(sGroup);
+            textView.setTextSize(30);
+            textView.setPadding(0,15,0,15);
+            //Set text style Bold
+//            textView.setTypeface(null, Typeface.BOLD);
+//            //Set text colour
+//            textView.setTextColor(Color.BLACK);
+            //Return View
+            return textView;
+
+        }
+
+        @Override
+        public View getChildView(int i, int i1, boolean b, View view, ViewGroup viewGroup) {
+            TextView textView = new TextView(my_caregivee.this.getActivity());
+            textView.setText(getChild(i, i1).toString());
+//            textView.setText(getGroup(i).toString());
+            //Initialize string
+            String sChild = String.valueOf(getChild(i, i1));
+            //Set text on text view
+            textView.setText(sChild);
+            textView.setPadding(0,10,0,10);
+            textView.setTextSize(20);
+            //Set text style Bold
+            //textView.setTypeface(null, Typeface.BOLD);
+            textView.setOnClickListener(v -> {
+                switch(i1) {
+                    case 0:
+                        ((Dashboard)getActivity()).replaceActiveFragment(new ProfileFragment());
```

Comment:
You are supposed to open to the profile of the caregivee! Not the caregiver. Report bug.